docs/02-contributing.md

```diff
@@ -4,6 +4,10 @@
 
 Please look at [Github Issues](https://github.com/awesomeWM/awesome/issues).
 
+## Triage Issues [![Open Source Helpers](https://www.codetriage.com/awesomewm/awesome/badges/users.svg)](https://www.codetriage.com/awesomewm/awesome)
+
+You can triage issues which may include reproducing bug reports or asking for vital information, such as version numbers or reproduction instructions. If you would like to start triaging issues, one easy way to get started is to [subscribe to awesome on CodeTriage](https://www.codetriage.com/awesomewm/awesome).
```

Comment:
s/awesome/AwesomeWM/

Reply:
Updated. I think you meant the "subscribe to awesome" part. I went ahead and changed the case on the rest of the links as well.